<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewPatientWindow</class>
 <widget class="QDialog" name="AddNewPatientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>947</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddNewPatientWindow</string>
  </property>
  <property name="windowIcon">
   <iconset theme="camera-web">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>901</width>
     <height>861</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>897</width>
      <height>857</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>880</width>
       <height>200</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QTextEdit" name="name_textEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="tel_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TEL:</string>
      </property>
     </widget>
     <widget class="QLabel" name="birthdata_label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Birth Date:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="remark_textEdit">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>60</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="address_label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Home Address:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="birth_dateEdit">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>60</y>
        <width>165</width>
        <height>42</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="locale">
       <locale language="Chinese" country="China"/>
      </property>
      <property name="displayFormat">
       <string>yyyy/MM/dd</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="currentSectionIndex">
       <number>0</number>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
      <property name="date">
       <date>
        <year>1999</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
     <widget class="QLabel" name="remark_label">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Remark:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="tel_textEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="gender_label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Gender:</string>
      </property>
     </widget>
     <widget class="QLabel" name="name_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="address_textEdit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>110</y>
        <width>541</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QComboBox" name="gender_comboBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>125</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <item>
       <property name="text">
        <string>(Select)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Patient Info&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="add_time_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add time:</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="add_dateTimeEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>226</width>
        <height>42</height>
       </rect>
      </property>
      <property name="locale">
       <locale language="Chinese" country="China"/>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>8</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>yyyy/MM/dd HH:mm:ssâ€¯</string>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>880</width>
       <height>240</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="medical_history_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Medical History&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="allergic_history_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Allergic History: </string>
      </property>
     </widget>
     <widget class="QTextEdit" name="allergic_history_textEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>651</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="past_medical_history_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Past Medical &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;History:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="past_medical_history_textEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>651</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </widget>
    <widget class="QScrollBar" name="verticalScrollBar">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>10</y>
       <width>16</width>
       <height>931</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>-60</x>
       <y>440</y>
       <width>1026</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>440</y>
       <width>880</width>
       <height>391</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="outpatient_records_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Outpatient Records&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="chief_complaint_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Chief Complaint:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="chief_complaint_textEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>651</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="history_of_the_present_illness_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>111</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;History of the&lt;/p&gt;&lt;p align=&quot;center&quot;&gt; Present &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Illness:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="history_of_the_present_illness_textEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>651</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="examinination_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Examination:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="diagnosis_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Diagnosis:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="examinination_textEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>651</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QTextEdit" name="diagnosis_textEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>651</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="remedy_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Remedy:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="remedy_textEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>651</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </widget>
    <zorder>groupBox_2</zorder>
    <zorder>verticalScrollBar</zorder>
    <zorder>line_2</zorder>
    <zorder>groupBox</zorder>
    <zorder>groupBox_3</zorder>
   </widget>
  </widget>
  <widget class="QPushButton" name="ok_pushButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>890</y>
     <width>103</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="print_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>880</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_pushButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>890</y>
     <width>103</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
